pubsub-lib/NatsClient.go

```diff
@@ -38,12 +38,18 @@ const DefaultMaxAge time.Duration = 86400000000000
 
 type NatsClientConfig struct {
 	NatsServerHost string `env:"NATS_SERVER_HOST" envDefault:"nats://devtron-nats.devtroncd:4222"`
+
 	//consumer wise
 	NatsMsgProcessingBatchSize int `env:"NATS_MSG_PROCESSING_BATCH_SIZE" envDefault:"1"`
 	NatsMsgBufferSize          int `env:"NATS_MSG_BUFFER_SIZE" envDefault:"64"`
+
 	//stream wise
 	NatsStreamConfig string `env:"NATS_STREAM_CONFIG" envDefault:"{\"max_age\":86400000000000}"`
+
+	// Consumer config
+	NatsConsumerConfig string `env:"NATS_CONSUMER_CONFIG" envDefault:"{\"ackWaitInSecs\":3600}"`
```

Comment:
can we move these numbers in constants or env variables. eg: 3600, 86400000000000